src/ufo.rs

```diff
@@ -419,6 +448,27 @@ fn validate_groups(groups_map: &Groups) -> Result<(), GroupsValidationError> {
     Ok(())
 }
 
+fn libs_to_object_libs(fontinfo: &Option<FontInfo>) -> Plist {
```

Comment:
ditto I think this should be a method on `FontInfo`, and then at the callsite you would do,

`self.font_info.map(FontInfo::libs_to_object_libs).unwrap_or_else(|| Plist::new())` or what-have-you.